Week1/homework/src/1-countries_8_million.js

```diff
@@ -0,0 +1,9 @@
+'use-strict';
+
+const select_function = require('./select-query-function');
+
+const sql_query = `SELECT name FROM country where population > 8;`;
+const message = 'Countries with population more than 8 million.';
+const column = 'name';
+
+select_function(sql_query, message, column);
```

Comment:
How about instead of separating these answers in different files, you make use of some kind of prompt library https://github.com/flatiron/prompt#readme ?

Reply:
I have made changes as requested. 
-made all queries in one file using the prompt module.